content/en/docs/concepts/security/index.md

```diff
@@ -632,331 +628,260 @@ authorization result, `ALLOW` or `DENY`.
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.headers[version]
+     values: ["v1", "v2"]
 {{< /text >}}
 
-#### `ServiceRoleBinding`
-
-A `ServiceRoleBinding` specification includes two parts:
-
--  **`roleRef`** refers to a `ServiceRole` resource in the same namespace.
--  A list of **`subjects`** that are assigned to the role.
-
-You can either explicitly specify a *subject* with a `user` or with a set of
-`properties`.  A *property* in a `ServiceRoleBinding` *subject* is similar to
-a *constraint* in a `ServiceRole` specification. A *property* also lets you use
-conditions to specify a set of accounts assigned to this role. It contains a
-`key` and its allowed *values*. The supported `key` values of a constraint
-are listed in the
-[constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).
-
-The following example shows a `ServiceRoleBinding` named
-`test-binding-products`, which binds two subjects to the `ServiceRole` named
-`"product-viewer"` and has the following `subjects`
+In the case that the attribute is a `map`, for example `request.headers`, the
+`key` is an entry in the map, for example `request.headers[version]`. The supported `key` values
+of a constraint are listed in the [conditions page](/docs/reference/config/authorization/conditions/).
 
-- A service account representing service **a**, `"service-account-a"`.
-- A service account representing the Ingress service
-  `"istio-ingress-service-account"` **and** where the JWT `email` claim is
-  `"a@foo.com"`.
+You can specify flexible source identities in the authorization policy. The following example
+shows an `AuthorizationPolicy` that allows two sources (service account `cluster.local/ns/default/sa/sleep`
+and namespace `dev`) to access the httpbin workload:
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRoleBinding
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: test-binding-products
-  namespace: default
+ name: httpbin
+ namespace: foo
 spec:
-  subjects:
-  - user: "service-account-a"
-  - user: "istio-ingress-service-account"
-    properties:
-      request.auth.claims[email]: "a@foo.com"
-  roleRef:
-    kind: ServiceRole
-    name: "products-viewer"
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   - source:
+       namespaces: ["dev"]
+   to:
+   - operation:
+       methods: ["GET", "POST"]
 {{< /text >}}
 
-In case you want to make a service publicly accessible, you can set the
-`subject` to `user: "*"`. This value assigns the `ServiceRole` to **all (both authenticated and
+In case you want to make a workload publicly accessible, you can set the
+`principals` to `principals: ["*"]`. This value allows sources from **all (both authenticated and
 unauthenticated)** users and services, for example:
```

Comment:
This is not right. If you set `principals: ["*"]`, it means principals field cannot be empty (authenticated users only). 

If you want to allow both authenticated and unauthenticated users, you need to leave "source" section empty.

Reply:
Updated the doc, we need to double check the implementation is doing this way.